src/lib.rs

```diff
@@ -78,14 +78,32 @@ pub mod context;
 pub mod shm;
 
 /// Initializes the zenoh runtime logger, using rust environment settings.
+/// E.g.: `RUST_LOG=info` will enable logging at info level. Similarly, you can set the variable to `error` or `debug`.
 ///
-/// Note that unless you built zenoh-c with the `logger-autoinit` feature disabled,
-/// this will be performed automatically by `z_open` and `z_scout`.
+/// Note that if the environment variable is not set, then logging will not be enabled.
 #[no_mangle]
-pub extern "C" fn zc_init_logging() {
+pub extern "C" fn zc_try_init_log_from_env() {
     zenoh::try_init_log_from_env();
 }
 
+/// Initializes the zenoh runtime logger, using rust environment settings or the provided fallback level.
+/// E.g.: `RUST_LOG=info` will enable logging at info level. Similarly, you can set the variable to `error` or `debug`.
+///
+/// Note that if the environment variable is not set, then fallback level will be used instead.
+///
+/// @param level: The fallback level for logging if the environment variable is not set.
```

Comment:
It is not a level it is a filter, right ? If we want only to allow level, it is probably better to use zc_log_severity_t and convert it to string inside the function call.

Reply:
It is a filter, correct.